srcpkgs/graphene/template

```diff
@@ -1,25 +1,25 @@
 # Template file for 'graphene'
 pkgname=graphene
-version=1.8.2
-revision=2
+version=1.10.0
+revision=1
 build_style=meson
 build_helper="gir"
 configure_args="-Dtests=false -Dbenchmarks=false
- -Dintrospection=$(vopt_if gir true false)"
+ -Dintrospection=$(vopt_if gir true false) -Darm_neon=false"
```

Comment:
I suspect you've disabled neon simply to unbreak the build on armv6, isn't it?

Reply:
If I remember correctly NEON is not architectural in armv6 and armv7, so apart of circumventing the error we can ensure compatibility with all armv* devices regardless of the extensions present on their CPUs.

Reply:
Neon was introduced with armv7 but support was optional. Void does not use neon for armv7 (with some exceptions).  So for armv6 and armv7 this is ok, but it should not be disabled for armv8.